droidlet/dashboard/web/src/components/Timeline/TimelineUtils.js

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright (c) Facebook, Inc. and its affiliates.
+*/
+
+import React from "react";
+
+export function capitalizeEvent(str) {
+  // replaces underscores with spaces
+  str = str.replace(/_/g, " ");
+  // capitalizes the first letter of every word
+  return str.replace(/\w\S*/g, function (txt) {
+    return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();
+  });
+}
+
+export function renderTable(tableArr) {
+  // returns an HTML table given an array
+  if (tableArr) {
+    return tableArr.map((data) => {
+      const { event, description } = data;
+      return (
+        <tr>
+          <td>
+            <strong>{event}</strong>
+          </td>
+          <td>{description}</td>
+        </tr>
+      );
+    });
+  }
+}
+
+export function jsonToArray(eventObj) {
+  // turns JSON hook data into an array that can easily be turned into an HTML table
+  let tableArr = [];
+  for (let key in eventObj) {
+    if (eventObj.hasOwnProperty(key)) {
```

Comment:
is there any chance to go inside the else condition here? If I understand correctly you are iterating over the keys in eventObj and check if key is in eventObj

Reply:
Great point, I don't think the if case is necessary here since there aren't any inherited properties. I'll remove this in the next commit